Wrappers/Python/ccpi/optimisation/operators/FiniteDifferenceOperator.py

```diff
@@ -315,7 +315,7 @@ def domain_geometry(self):
         '''Returns the domain geometry'''
         return self.gm_domain
        
-    def norm(self):
+    def calculate_norm(self):
```

Comment:
Why the need for a dedicated `calculate_norm` for `FiniteDifferenceOperator`?  Wouldn't the standard inherited one work?

Reply:
To my understanding there is not standard method of norm in any class. Only concrete classes have this method implemented. 

Maybe your suggestion is to add the *standard* `PowerMethod` as method to calculate the `norm` of a `LinearOperator`?

Reply:
Yes, I thought that was the case. Indeed let us have the power method used in `calculate_norm` in the `LinearOperator`. Then for any subclasses we only need to implement `calculate_norm` if we have a specific more efficient version, like for the identity.